third_party/tsl/tsl/platform/cpu_info.cc

```diff
@@ -23,10 +23,14 @@ limitations under the License.
 #include <mutex>  // NOLINT
 #endif
 #if defined(PLATFORM_IS_ARM64) && !defined(__APPLE__) && !defined(__OpenBSD__)
+#include <asm/hwcap.h> /* Get HWCAP bits from asm/hwcap.h */
 #include <sys/auxv.h>
 #ifndef HWCAP_CPUID
 #define HWCAP_CPUID (1 << 11)
 #endif
+#ifndef ARM_COMPUTE_CPU_FEATURE_HWCAP2_BF16
+#define ARM_COMPUTE_CPU_FEATURE_HWCAP2_BF16 (1 << 14)
+#endif
```

Comment:
Preprocessor defines are discouraged. Please make them `static constexpr` class members instead. Please follow this [constant naming guideline](https://google.github.io/styleguide/cppguide.html#Constant_Names).